share/spice/yoga_asanas/yoga_asanas.js

```diff
@@ -0,0 +1,49 @@
+(function(env) {
+    "use strict";
+
+    env.ddg_spice_yoga_asanas = function(api_result){
+
+        if (!api_result || !api_result.response.numFound) {
+            return Spice.failed('yoga_asanas');
+        }
+
+        var docs = api_result.response.docs,
```

Comment:
We should run `JSON.parse()` here so we don't need to make any subsequent calls to parse the data.